### DISCUSSION TITLE
Help With Gaining Maximum Read Speed

### DISCUSSION THREAD

**Discussion by jakanaka** [trapexit/mergerfs#1293] [General]
I have a zfs pool on /mnt/tank and md raid on /mnt/spool. I have my media content there. The issue i face is the mergerfs mount point gets very slow on high io operations with default settings. I read the readme and tuned the options like this now

`mergerfs /mnt/tank=RO:/mnt/spool=RO /mnt/media -o security_capability=false,xattr=noattr,nonempty,allow_other,use_ino,cache.files=off,cache.readdir=true,cache.symlinks=true,symlinkify=true,symlinkify_timeout=0,fsname=media,threads=8
`

this seem to have got this to at least a usable state bt the original fs is more faster. i don't write at at the the mountpoint. they are all video and static content. any help on how to achieve maximum read speed and io operations

**Comment by trapexit** [comment]
If you are using symlinkify then you aren't even using mergerfs for reads.

**Reply by trapexit** [reply]
There is really nothing additional to say besides what I have in the docs. Unless you provide more information there really isn't much to say. I have sections on support and benchmarking which gives you a bunch of details to share.

**Reply by jakanaka** [reply]
symlinkify_timeout=0 means everything will be symlinked directly, right? can you suggest any other flags i should add or change?

**Reply by trapexit** [reply]
It means as it mentions. That once the current time - mtime exceeds that value it will be symlinked. And the file must be read only.

As I said... I've nothing to add without more info / context. If I had anything additional generally to suggest it would already be in the docs.
